explainaboard/utils/serialization.py

```diff
@@ -4,11 +4,14 @@
 import dataclasses
 from inspect import getsource
 
+from explainaboard.analysis.feature import FeatureType, get_feature_type_serializer
 from explainaboard.utils.tokenizer import get_tokenizer_serializer, Tokenizer
 
 
 def general_to_dict(data):
     """DEPRECATED: do not use this function for new implementations."""
+    if isinstance(data, FeatureType):
```

Comment:
It seems that we have: `explainaboard/utils/serialization` and `explainaboard/serialization`, do we consider putting them together?

Reply:
Moved it to `explainaboard.serialization.legacy`